archive/cpio_old_le.ksy

```diff
@@ -0,0 +1,64 @@
+meta:
+  id: cpio_old_le
+  title: cpio archive, old binary variant, little-endian
+  file-extension: cpio
+  license: CC0-1.0
+  endian: le
+  encoding: ASCII

```

Comment:
It was really only required to make the "path_name" field more accessible/usable (as opposed to displaying a byte array). Happy to change to a byte array if that is better, as there is never going to be a valid "encoding" for the path name (given the character set is file-system specific)